src/components/SideNavBar/index.tsx

```diff
@@ -80,13 +96,11 @@ export default function AboutPageSideNavBar({
   }
 
   const navElement = useRef<HTMLElement | null>(null);
-  return (
-    <nav className={className} ref={navElement}>
-      <button type="button" className="side-nav__open" onClick={toggle}>
-        Menu
-      </button>
-      <ul className="community-nav__list">
-        {aboutPageSideNavBarItem.map(({ title: commTitle, slug }) => {
+
+  const generateNavItems = () => {
```

Comment:
Completely open to other ideas to dynamically generate navbaritems. I went with switch in case we have similar pages.